arbeitszeit_web/www/controllers/edit_draft_controller.py

```diff
@@ -0,0 +1,89 @@
+from dataclasses import dataclass
+from uuid import UUID
+
+from arbeitszeit.use_cases.edit_draft import Request
+from arbeitszeit_web.forms import DraftForm
+from arbeitszeit_web.notification import Notifier
+from arbeitszeit_web.session import Session
+from arbeitszeit_web.translator import Translator
+
+
+@dataclass
+class EditDraftController:
+    notifier: Notifier
+    translator: Translator
+    session: Session
+
+    def process_form(self, form: DraftForm, draft_id: UUID) -> Request | None:
+        product_name = form.product_name_field().get_value()
+        if not product_name:
+            form.product_name_field().attach_error(
+                self.translator.gettext("The product name field cannot be empty.")
```

Comment:
Just FYI: These error messages currently are not visible to the user (at least in the browsers I tested). Instead, a message like "Fülle dieses Feld aus" (Chrome) or "Bitte füllen Sie dieses Feld aus" (Firefox) appears.